tests/conformance/conformance_test.go

```diff
@@ -3003,6 +3003,22 @@ var internalTestCases = []testCase{
 		dockerUseBuildKit: true,
 	},
 
+	{
+		name:              "heredoc-copy",
+		dockerfile:        "Dockerfile.heredoc_copy",
+		dockerUseBuildKit: true,
+		contextDir:        "heredoc",
+		fsSkip: []string{"(dir):test:mtime",
+			"(dir):test2:mtime",
+			"(dir):test:(dir):humans.txt:mtime",
+			"(dir):test:(dir):robots.txt:mtime",
+			"(dir):test:(dir):file:mtime",
```

Comment:
This file comes from the build context, so I'd expect its datestamp to be consistent between different builds.

Reply:
correct, this skip is not needed.